src/scss/buttons.scss

```diff
@@ -248,3 +248,7 @@ button.no-btn {
     color: themed("textSuccessColor");
   }
 }
+
+.org-name-badge {
+  width: 100px;
+}
```

Comment:
Please use tailwind for this, `tw-w-24`. We shouldn't be writing any new css at all. https://tailwindcss.com/docs/width